force-app/main/default/classes/PMT_RefundController.cls

```diff
@@ -134,16 +134,14 @@ public with sharing class PMT_RefundController {
             return;
         }
 
-        refundService.updateParentOpportunity(originalPayment, refundAmount);
```

Comment:
I just want to be sure: this Opportunity logic is now being run by the trigger that's called after `refundService.insertRefunds();`?